formats/protobuf/commonMain/src/kotlinx/serialization/SerialId.kt

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package kotlinx.serialization
+
+
+@SerialInfo
+@Target(AnnotationTarget.PROPERTY)
+@Deprecated(message = "SerialId is renamed to ProtoId to better reflect its semantics",
```

Comment:
Can we communicate somehow that protobuf was extracted to separate module?

Reply:
We can add `SerialId` back to the `runtime` module with a message that mentions that, but then its deprecation level should be increased to `ERROR`.
Adding anything else in `runtime` module is a no-go, because it will seriously complicate UX when protobuf module is already included

Reply:
Yeah, I think it's better with message like "SerialId is deprecated in favor of ProtoId from kotlinx-serialization-protobuf module"